haml_lint.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.required_ruby_version = '>= 1.9.3'
 
   s.add_dependency 'haml', '>= 4.0', '< 5.1'
+  s.add_dependency 'rainbow'
```

Comment:
We're using the Rainbow gem, so we should probably list it as a dependency instead of relying on the transitive inclusion through Rubocop.